config/routes.rb

```diff
@@ -1,3 +1,10 @@
 Rails.application.routes.draw do
-  root controller: :product, action: :index
+  get 'products/show'
+  devise_for :users
+
+  get 'landing/index'
+  
+  resources :products
```

Comment:
please clean this file up from redundant routes

Reply:
Changed according to comments.